### ISSUE TITLE
add cli command for manually approving a piece of freight for a given stage

### ISSUE THREAD

**Issue by krancour** [akuity/kargo#1074]
Follow up to #1073

**Comment by krancour** [comment]
#1162 closed this in error. Re-opening.

**Comment by krancour** [comment]
This really depends on #1222 to function correctly. #1222 has been deferred to v0.4.0, so this must be also.
